### ISSUE TITLE
RZFOv15

### ISSUE THREAD

**Issue by 61Stan** [eurosecom/projekt1#102]
Ročné zúčtovanie preddavkov na daň z príjmov fyzickej osoby zo závislej činnosti pre rok 2015

**Comment by eurosecom** [comment]
"na edcom.sk všetky RZ dáva (tlačiareň v záhlaví form) pdf na 1 stranu"  znamená že nedáva podklad JPG na 1.stranu alebo nie je ikonka na tlač do PDF ?
Lebo ja mám aj aj v IE aj Chrome.

**Comment by 61Stan** [comment]
Keď tlačím RZ (ikona tlačiarne v záhlaví formulára = vedľa boxu s Preniesť) tak u všetkých tlačí na 1 stranu. Vydedukoval som, že 2.stranu tlačí, keď sú nejaké úpravy na 2. strane. Ale keď dám tlačiť RZ za Andrej Dobiaš, tak mi dáva pdf:
[rocnedane.54.pdf](https://github.com/eurosecom/projekt1/files/161867/rocnedane.54.pdf)
Ale ja tam žiadne úpravy nevidím. Alebo stačí vojsť do úpravy a stlačiť "uložiť zmeny" a tým pádom akokeby som niečo robil na 2.strane.
Robí mi to aj na druhom počítači.

**Comment by eurosecom** [comment]
Áno, keď tam užívateľ vbehne a uloží zmeny tak bude tlačiť 2.stranu
